src/test/java/racing/model/RacingGameTest.java

```diff
@@ -53,9 +57,7 @@ void move() {
         racingGame.moveAndStop(randomValue);
 
         //then
-        assertEquals(5, racingCar.get(0).getPosition());
-        assertEquals(7, racingCar.get(1).getPosition());
-        assertEquals(8, racingCar.get(2).getPosition());
+        assertThat(racingCar).allSatisfy(car -> assertEquals(car.getPosition(), 1));
```

Comment:
테스트가 간결해 졌네요! 👍 
```suggestion
        assertThat(racingCar).allSatisfy(car -> assertEquals(1, car.getPosition()));
```
`assertEquals` 의 경우에는 기대값이 먼저 옵니다. 헷갈리죠? 그래서 왠만하면 junit 단언문을 섞어쓰지 않고 assertj 단언문으로만 사용합니다.